srcpkgs/open-iscsi/files/iscsid/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+exec 2>&1
+[ -r conf ] && . ./conf
```

Comment:
what's the point of this source when we are not using any variables from it?

Reply:
Good point, I'll remove it.

Reply:
done.

Reply:
Did you forget to commit or push your removal of this line? It's still there.

Reply:
oops, forgot to add, re-check now please.